sdk/src/main/scala/coop/rchain/sdk/primitive/VoidSyntax.scala

```diff
@@ -0,0 +1,13 @@
+package coop.rchain.sdk.primitive
+
+trait VoidSyntax {
+  implicit def sdkSyntaxVoid[A](a: A): VoidOps[A] = new VoidOps(a)
+}
+
+final class VoidOps[A](private val a: A) extends AnyVal {
+
+  /**
+    * Ignore value and return [[Unit]].
+    */
+  def void(): Unit = ()
```

Comment:
Convenient way for exceptional cases where we want to ignore a value and have statement instead of expression.

**Also dangerous because hides potential unused value. Please use sparingly.**

Reply:
> Also dangerous because hides potential unused value. Please use sparingly.

Make sense to add TODO comment directly in the code.